src/test/java/picard/illumina/parser/TileMetricsUtilTest.java

```diff
@@ -80,4 +103,27 @@ private Path createTestDirs(int numCycleMetricsFiles, boolean baseMetricsFile) t
         }
         return baseDir;
     }
+
+    private List<File> generateFindTileMetricsExpected(Path baseDir, List<Integer> populatedCycleDirs, boolean baseMetricsFile) {
+        List<File> expected = new ArrayList<>();
+        if (baseMetricsFile) {
+            expected.add(new File(
+                baseDir
+                    .resolve(TileMetricsUtil.INTEROP_SUBDIRECTORY_NAME)
+                    .resolve(TileMetricsUtil.TILE_METRICS_OUT_FILE_NAME).toUri())
```

Comment:
why `new File(<path expression>.toUri()` instead of `<path expression>.toFile()`?